src/main/scala/tech/cryptonomic/conseil/tezos/TezosOptics.scala

```diff
@@ -120,19 +162,44 @@ object TezosOptics {
           transactionBigMapDiffs composeTraversal
           (Traversal.fromTraverse[List, Contract.CompatBigMapDiff] composeOptional selectBigMapRemove)
 
-    val readBigMapDiffCopy = blockOperationsGroup composeTraversal groupOperations composeTraversal operationBigMapDiffCopy
+    //planned for reuse
+    private val script = GenLens[Origination](_.script)
```

Comment:
can you suppress warnings for this and other lenses before making this PR final if you not to plan to use them this time?

Reply:
will do